### DISCUSSION TITLE
What is the cause of "The XXX organization cannot be invited because it has marketplace applications"?

### DISCUSSION THREAD

**Discussion by OndrejSpanel** [community/community#139255] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I want to start a trial of enterprise. When doing so, I am told "The XXX organization cannot be invited because it has marketplace applications."

I am not aware of any. I had two private GitHub applications in this organization, but even when I moved them to a different account, nothing has changed. How can I find what "marketplace applications" are blocking the trial?

**Comment by OndrejSpanel** [comment, marked as answer]
I have found an answer in https://docs.github.com/en/billing/managing-billing-for-github-marketplace-apps/canceling-a-github-marketplace-app

I had activated a free subscription of one app long time ago and forgotten about it, because I was not using the app.

**Reply by queenofcorgis** [reply]
@OndrejSpanel nice self-solve ✅  thanks for coming back to share the answer, this helps others who may have the same question as you.
